### ISSUE TITLE
Existing Container with same PartitionKey Path throwing error

### ISSUE THREAD

**Issue by tmeadlin** [Azure/azure-cosmos-dotnet-v3#652]
**Describe the bug**
I have existing containers that were built using Microsoft.Azure.DocumentDB.Core v2.4.1. I've updated my code to be using Microsoft.Azure.Cosmos v3.1.0. I would not think I would need to use the migration tool for this as this is still Cosmos. Upon running the code below, I receive the following error.

_The requested partition key path '/partition_key' does not match existing Container 'StaticData' with partition key path '/partition_key'
Parameter name: PartitionKey_

**Code to Create Container in Microsoft.Azure.Cosmos v3.1.0**
```
var containerProperties = new ContainerProperties(cosmosContainer.Name, cosmosContainer.PartitionKey)
{
  DefaultTimeToLive = cosmosContainer.DefaultTimeToLive > 0
      ? (int)TimeSpan.FromDays(cosmosContainer.DefaultTimeToLive.Value).TotalSeconds
      : cosmosContainer.DefaultTimeToLive
};

 return await _databaseResponse.Database.CreateContainerIfNotExistsAsync(containerProperties);
```

**Code to Create Collection in Microsoft.Azure.DocumentDB.Core v2.4.1**
```
return _client.CreateDocumentCollectionIfNotExistsAsync(
  UriFactory.CreateDatabaseUri(databaseName),
  new DocumentCollection
  {
    Id = cosmosCollection.Name,
    PartitionKey = new PartitionKeyDefinition {Paths = cosmosCollection.PartitionKeys},
    DefaultTimeToLive = cosmosCollection.DefaultTimeToLive > 0
        ? (int)TimeSpan.FromDays(cosmosCollection.DefaultTimeToLive.Value).TotalSeconds
        : cosmosCollection.DefaultTimeToLive
});
```

**Expected behavior**
I would expect the code to recognize that these are the same partionKeyPaths. If this is something that requires usage of the Migration Tool, I would hope the exception would be a little more clear. As of right now the exception basically says that these paths don't match but clearly in the message they do.

**Environment summary**
SDK Version: 3.1.0
OS Version: Windows 10 Pro

**Comment by j82w** [comment]
@simplynaveen20 will this be fixed by PR #629?

**Comment by simplynaveen20** [comment]
Yes is taken care in #629 , will be available in next release
